metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java

```diff
@@ -111,10 +116,26 @@
     public static final String SECOND_ISBN_ISBN13_VARIANT = "9781627050128";
     public static final String ONLINE_ISSN = "2052-2916";
     public static final String PRINT_ISSN = "0969-0700";
-    public static final String ENSURE_ONLY_FILTERING_TITLES = "McLongName, Longy (Colonel (ret'd))";
     private static final TestAppender logger = LogUtils.getTestingAppenderForRootLogger();
     private WireMockServer wireMockServer;
 
+    @Test
+    public void getJournalIdFromPublicationChannelsByGivenJournalName()
```

Comment:
This test follows the old pattern we used in fetch-doi, we should have a discussion about refactoring this — set up a task for that.